### DISCUSSION TITLE
Twitter Invalid Credentials?

### DISCUSSION THREAD

**Discussion by daffatahtaa** [AAndyProgram/SCrawler#132] [Q&A] [answered]
Hello, 

I have a problem while downloading on twitter, the log said:
```
2023-05-30 04:04:46: Twitter has invalid credentials
```

here's my cookie settings

![image](https://github.com/AAndyProgram/SCrawler/assets/43267539/57212f5c-b30b-4418-9913-fcf81dbc4534)

Please help,
Thanks

**Comment by AAndyProgram** [comment, marked as answer]
I think the problem is the number of cookies. Why are you adding so few cookies?

**Reply by daffatahtaa** [reply]
wait lemme try again
